pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/make-bootstrap-sources.nix

```diff
@@ -9,23 +8,30 @@
 #
 # To build:
 #
-#   nix-build pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/make-bootstrap-sources.nix
+#   nix-build '<nixpkgs>' -A minimal-bootstrap-sources
```

Comment:
Please change this to the new way of updating the narball hash :)

Reply:
a877fb253db2